examples/controlnet/train_controlnet.py

```diff
@@ -25,6 +25,7 @@
 import torch
 import torch.nn.functional as F
 import torch.utils.checkpoint
+import torchvision.transforms.functional as TF
```

Comment:
Funny alias, though 😂

Reply:
BTW not a huge fan of single letter abbreviations.

```py
from torchvision.transforms.functional import resize
```

is cleaner IMO

same with just using:
```py
torch.nn.functional.gelu(...)
```
is better IMO then
```py
F.gelu(...)
```
But just a nit! Ok for me